.github/workflows/ci.yaml

```diff
@@ -35,7 +35,9 @@ jobs:
         run: cargo clippy -- -D warnings
 
       - name: Build and Test
-        run: cargo test --features=apple-native,windows-native --verbose
+        env:
+          RUST_LOG: debug
+        run: cargo test --features=apple-native,windows-native --verbose -- --nocapture
```

Comment:
why `--nocapture`? You don't seem to be verifying the logs. In a CI context, this seems to make the logs unnecessarily long.

IIRC, even without --nocapture, if a test fails then the debug log output will be shown for that test.

Reply:
It was to force them to show, just a suggestion. I removed it.